sonar-orchestrator/src/main/java/com/sonar/orchestrator/build/SonarScanner.java

```diff
@@ -0,0 +1,181 @@
+    return (SonarScanner) super.setRunnerVersion(s);
+  }
+
+  @Override
+  public SonarScanner setProjectDir(File dir) {
+    return (SonarScanner) super.setProjectDir(dir);
+  }
+
+  @Override
+  public SonarScanner setProjectKey(@Nullable String s) {
+    return (SonarScanner) super.setProjectKey(s);
+  }
+
+  @Override
+  public SonarScanner setProjectVersion(@Nullable String s) {
+    return (SonarScanner) super.setProjectVersion(s);
+  }
+
+  @Override
+  public SonarScanner setProjectName(@Nullable String s) {
+    return (SonarScanner) super.setProjectName(s);
+  }
+
+  @Override
+  public SonarScanner setSourceDirs(@Nullable String s) {
+    return (SonarScanner) super.setSourceDirs(s);
+  }
+
+  @Override
+  public SonarScanner setTestDirs(@Nullable String testDirs) {
+    return (SonarScanner) super.setTestDirs(testDirs);
+  }
+
+  /**
+   * @deprecated since 3.8 sonar.binaries is deprecated
+   */
+  @Deprecated
+  @Override
+  public SonarScanner setBinaries(@Nullable String s) {
+    return (SonarScanner) super.setBinaries(s);
+  }
+
+  /**
+   * @deprecated since 3.8 sonar.libraries is deprecated
+   */
+  @Deprecated
+  @Override
+  public SonarScanner setLibraries(@Nullable String s) {
+    return (SonarScanner) super.setLibraries(s);
+  }
+
+  @Override
+  public SonarScanner setLanguage(@Nullable String s) {
+    return (SonarScanner) super.setLanguage(s);
+  }
+
+  @Override
+  public SonarScanner setSourceEncoding(@Nullable String s) {
+    return (SonarScanner) super.setSourceEncoding(s);
+  }
+
+  @Override
+  public SonarScanner setDebugLogs(boolean b) {
+    return (SonarScanner) super.setDebugLogs(b);
+  }
+
+  @Override
+  public SonarScanner setShowErrors(boolean b) {
+    return (SonarScanner) super.setShowErrors(b);
+  }
+
+  @Override
+  public SonarScanner setTask(String task) {
+    return (SonarScanner) super.setTask(task);
+  }
+
+  @Override
+  public SonarScanner setProfile(String profileKey) {
+    return (SonarScanner) super.setProfile(profileKey);
+  }
+
+  @Override
+  public SonarScanner addArgument(String additionalArgument) {
+    return (SonarScanner) super.addArgument(additionalArgument);
+  }
+
+  @Override
+  public SonarScanner addArguments(List<String> additionalArguments) {
+    return (SonarScanner) super.addArguments(additionalArguments);
+  }
+
+  @Override
+  public SonarScanner addArguments(String... additionalArguments) {
+    return (SonarScanner) super.addArguments(additionalArguments);
+  }
+
+  @Override
+  public SonarScanner setProperty(String key, String value) {
+    return (SonarScanner) super.setProperty(key, value);
+  }
+
+  @Override
+  public SonarScanner setProperties(Map<String, String> p) {
+    return (SonarScanner) super.setProperties(p);
+  }
+
+  @Override
+  public SonarScanner setProperties(String... keyValues) {
+    return (SonarScanner) super.setProperties(keyValues);
+  }
+
+  public static SonarScanner create() {
+    return (SonarScanner) new SonarScanner()
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unnecessary cast to "SonarScanner". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS1905)